CONTRIBUTING.md

```diff
@@ -0,0 +1,36 @@
+# Contributing to Lumina
+
+First off, thanks for taking the time to contribute! ❤️ We appreciate all the efforts to help both Lumina and the Celestia ecosystem.
+
+
+Below are a few requirements we have for incoming submissions.
+
+## Signed Commits
+
+All commits need to be signed before PR can be merged, to ensure integrity
+and authenticity of the submitted code. If you don't have git signing set up,
+see [github's guide](https://docs.github.com/en/authentication/managing-commit-signature-verification/signing-commits) 
+for reference.
+
+## Conventional Commits
+
+We follow [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
+specification when writing commits messages. Consistency helps with readability
+of git history, additionally we use it to generate our changelog.
+
+## Upgrading dependencies (should you need to)
```

Comment:
Isn't "If you need to" more correct?

Reply:
afaik both are correct https://ludwig.guru/s/should+you+need

Reply:
I prefer "if you need to" as it's simpler to understand for not so expert english speakers like me :p

Reply:
i also prefer "if you need to"

Reply:
done